app/shared/components/Wallet/Panel/Unlocked.js

```diff
@@ -13,16 +12,28 @@ import WalletPanelButtonTransferSend from './Button/Transfer/Send';
 
 import WalletPanelButtonRamSell from './Button/Ram/Sell';
 import WalletPanelButtonRamBuy from './Button/Ram/Buy';
+import WalletPanelButtonWaxClaim from './Button/Wax/Claim';
 
 class WalletPanelUnlocked extends Component<Props> {
-  state = { activeIndex: 0 }
+  state = { activeIndex: 0 };
+
+  componentDidMount() {
+    const { actions, connection, settings } = this.props;
+
+    const isWaxChain =
+      connection.chainId === '1064487b3cd1a897ce03ae5b6a865651747e2e152090f99c1d19d44e01aea5a4';
```

Comment:
You might as well set this `isWaxChain` as a state variable, and then update it whenever the user changes between blockchains as well as on mount (like above). Then you could use the state value below on line 61.

Basically all I'm trying to do is have chain IDs defined in less places, should we have to find/replace :)

Reply:
Yeah good idea :D

Reply:
Done! Thanks for pointing that out!